tracker/dmlc_tracker/opts.py

```diff
@@ -141,12 +164,14 @@ def get_opts(args=None):
                         directory into remote machines\'s SYNC_DST_DIR')
     parser.add_argument('command', nargs='+',
                         help='Command to be launched')
+
```

Comment:
nit: do we need these additional blank lines ?

Reply:
removed